orc8r/tools/ansible/roles/pkgrepo/tasks/main.yml

```diff
@@ -48,20 +37,25 @@
   vars:
     packages:
       - ca-certificates
+  when: preburn
```

Comment:
I believe we can completely remove this step. I guess this was once necessary to do during `when: provision` when our VM images were really old. I assume that during preburn this shouldn't usually have an effect.

Reply:
Removed.